apps/scan/frontend/src/components/audio_diagnostic_modal_button.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { DiagnosticOutcome } from '@votingworks/types';
+import { Button, Icons, Modal, P } from '@votingworks/ui';
+import { useSound } from '../utils/use_sound';
+import * as api from '../api';
+
+export interface AudioDiagnosticModalProps {
+  setVisible: (visible: boolean) => void;
+}
+
+export function AudioDiagnosticModal(
+  props: AudioDiagnosticModalProps
+): JSX.Element {
+  const { setVisible } = props;
+  const playSound = useSound('success');
+  React.useEffect(playSound, [playSound]);
```

Comment:
I've been under the impression that we're moving away from `useEffect` when possible, instead triggering things on user action (clicking of the button / opening of the modal). I truly do not care, but noting for the future / alignment across engineers.

Reply:
Ah, I hadn't hard that - worth a team discussion, because not sure I can get behind a blanket approach without the full context (there are a few patterns we use that are equivalent to "do something when this component loads", like our data fetching, or screen reader logic). +1 to being consistent across, though